control-plane/agents/core/src/core/reconciler/replica/mod.rs

```diff
@@ -23,7 +26,10 @@ impl ReplicaReconciler {
 #[async_trait::async_trait]
 impl TaskPoller for ReplicaReconciler {
     async fn poll(&mut self, context: &PollContext) -> PollResult {
-        destroy_orphaned_replicas(context).await
+        let mut results = vec![];
+        results.push(disown_missing_owners(context).await);
+        results.push(destroy_orphaned_replicas(context).await);
+        Self::squash_results(results)
```

Comment:
I wonder if the pollTimer here should be set to 5 rather than 1, similar to what the volume does for garbage collection.

Reply:
Done.